arkane/statmech.py

```diff
@@ -1220,79 +1220,73 @@ def projectRotors(conformer, F, rotors, linear, is_ts, label):
-                 'mp2_rmp2_pvqz': 0.962,  # http://cccbdb.nist.gov/vibscalejust.asp, taken as ',p2/cc-pvdz'
-                 'ccsd-f12/cc-pvdz-f12': 0.947,  # http://cccbdb.nist.gov/vibscalejust.asp, taken as ccsd/cc-pvdz
-                 # 'ccsd(t)-f12/cc-pvdz-f12_noscale': ,
-                 # 'g03_pbepbe_6-311++g_d_p': ,
-                 # 'fci/cc-pvdz': ,
-                 # 'fci/cc-pvtz': ,
-                 # 'fci/cc-pvqz': ,
-                 # 'bmk/cbsb7': ,
-                 # 'bmk/6-311g(2d,d,p)': ,
-                 'b3lyp/6-31g**': 0.961,  # http://cccbdb.nist.gov/vibscalejust.asp
-                 'b3lyp/6-311+g(3df,2p)': 0.967,  # http://cccbdb.nist.gov/vibscalejust.asp
-                 'wb97x-d/aug-cc-pvtz': 0.974,
-                 # Taken from https://comp.chem.umn.edu/freqscale/version3b2.htm as ωB97X-D/maug-cc-pVTZ
+    freq_dict = {'hf/sto-3g': 0.817,  # [2]
+                 'hf/6-31g': 0.903,  # [2]
+                 'hf/6-31g(d)': 0.899,  # [2]
+                 'hf/6-31g(d,p)': 0.903,  # [2]
+                 'hf/6-31g+(d,p)': 0.904,  # [2]
+                 'hf/6-31+g(d,p)': 0.915 * 1.014,  # [1] Table 7
+                 'pm3': 0.940 * 1.014,  # [1] Table 7, the 0.940 value is the ZPE scale factor
+                 'pm6': 1.078 * 1.014,  # [1] Table 7, the 1.078 value is the ZPE scale factor
+                 'b3lyp/6-31g(d,p)': 0.961,  # [2]
+                 'b3lyp/6-311g(d,p)': 0.967,  # [2]
+                 'b3lyp/6-311+g(3df,2p)': 0.967,  # [2]
+                 'b3lyp/6-311+g(3df,2pd)': 0.970,  # [2]
+                 'm06-2x/6-31g(d,p)': 0.952,  # [2]
+                 'm06-2x/cc-pvtz': 0.955,  # [2]
+                 'm06-2x/aug-cc-pvtz': 0.985,  # [1] Table 3, [3]
+                 'm06-2x/def2-tzvpp': 0.983,  # [1] Table 3, [3]
+                 'm08so/mg3s*': 0.995,  # [1] Table 3, taken as 'M08-SO/MG3S'
+                 'wb97x-d/aug-cc-pvtz': 0.988,  # [3], taken as 'ωB97X-D/maug-cc-pVTZ'
+                 'wb97xd/6-311++g(d,p)': 0.992,  # calculated by Duminda [4]
+                 'mp2_rmp2_pvdz': 0.953,  # [2], taken as 'MP2/cc-pVDZ'
+                 'mp2_rmp2_pvtz': 0.950,  # [2], taken as 'MP2/cc-pVTZ'
+                 'mp2_rmp2_pvqz': 0.962,  # [2], taken as 'MP2/cc-pVQZ'
+                 'cbs-qb3': 0.99 * 1.014,  # [5], the 0.99 value is the ZPE scale factor
+                 'cbs-qb3-paraskevas': 0.99 * 1.014,  # [5], the 0.99 value is the ZPE scale factor
+                 'ccsd-f12/cc-pvdz-f12': 0.947,  # [2], taken as 'CCSD/cc-pVDZ'
+                 'ccsd(t)/cc-pvdz': 0.979,  # [2]
+                 'ccsd(t)/cc-pvtz': 0.975,  # [2]
+                 'ccsd(t)/cc-pvqz': 0.970,  # [2]
+                 'ccsd(t)/aug-cc-pvdz': 0.963,  # [2]
+                 'ccsd(t)/aug-cc-pvtz': 1.001,  # [3]
+                 'ccsd(t)/aug-cc-pvqz': 0.975,  # [2]
+                 'ccsd(t)/cc-pv(t+d)z': 0.965,  # [2]
+                 'ccsd(t)-f12/cc-pvdz-f12': 0.979,  # [2], taken as 'CCSD(T)/cc-pVDZ'
```

Comment:
You can update this line and the remaining lines using the newest version of [Truhlar's database](https://comp.chem.umn.edu/freqscale/190107_Database_of_Freq_Scale_Factors_v4.pdf). It has data for CCSD(T)-F12/cc-pVDZ-F12, CCSD(T)-F12/cc-pVTZ-F12 (already have this), and CCSD(T)-F12/cc-pVQZ-F12 (two different versions but they're super close to each other). The CCSD(T)-F12/aug-cc-pVnZ scale factors can be based on the CCSD(T)-F12/cc-pVnZ-F12 factors because the cc-pVnZ-F12 and aug-cc-pVnZ basis sets are very similar to each other.